README.md

```diff
@@ -1 +1,142 @@
-# Leaderboard-list-app-project
\ No newline at end of file
+# Leaderboard-list-app
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard-list-app-project] <a name="about-project"></a>
+
+**[Leaderboard-list-app-project]** is a simple app built with HTML and CSS to list scores.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+Client
+
+Key Features
+
+**[Leaderboard-setup-branch]**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Leaderboard-setup-branch]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- See live here : ().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ to use Git, Webpack and linters.
+ vs code
+```
+
+### Setup
+
+Clone this repository to your desired folder: (https://github.com/HERMON-1995/Leaderboard-list-app-project)
+
+```sh
+  cd my-folder
+  git clone [https://github.com/HERMON-1995/Leaderboard-list-app-project.git]
+```
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+- (Deployment method using GitHub pages)
+
```

Comment:
- [ ] Kindly update these README sections with some information instead of leaving them blank.